pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -269,6 +285,11 @@ buildStdenv.mkDerivation ({
     # 60.5+ & 66+ did split the google API key arguments: https://bugzilla.mozilla.org/show_bug.cgi?id=1531176
     configureFlagsArray+=("--with-google-location-service-api-keyfile=$TMPDIR/ga")
     configureFlagsArray+=("--with-google-safebrowsing-api-keyfile=$TMPDIR/ga")
+  '') + (lib.optionalString (ltoSupport && pgoSupport) ''
+    configureFlagsArray+=("--enable-lto=cross")
+    configureFlagsArray+=("--enable-profile-use=cross")
+    configureFlagsArray+=("--with-pgo-jarlog=${pgo}/en-US.log")
+    configureFlagsArray+=("--with-pgo-profile-path=${pgo}/merged.profdata")
   '') + ''
```

Comment:
```suggestion
  '') + lib.optionalString (ltoSupport && pgoSupport) ''
    configureFlagsArray+=("--enable-lto=cross" "--enable-profile-use=cross" "--with-pgo-jarlog=${pgo}/en-US.log" "--with-pgo-profile-path=${pgo}/merged.profdata")
  '' + ''
```

Reply:
This breaks readability big time, please don't.

Reply:
```suggestion
  '') + lib.optionalString (ltoSupport && pgoSupport) ''
    configureFlagsArray+=(
      "--enable-lto=cross"
      "--enable-profile-use=cross"
      "--with-pgo-jarlog=${pgo}/en-US.log"
      "--with-pgo-profile-path=${pgo}/merged.profdata"
    )
  '' + ''
```

Reply:
An even better solution is to just add those flags to configureFlags: https://github.com/Atemu/nixpkgs/commit/0d94218207d76299d5f44d41bdcd09007d86ac77.